deploy/helm/kube-cleanup-operator/README.md

```diff
@@ -0,0 +1,59 @@
+# kube-cleanup-operator
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 0.8.1](https://img.shields.io/badge/AppVersion-0.8.1-informational?style=flat-square)
+
+Kubernetes Operator to automatically delete completed Jobs and their Pods
+
+## Source Code
+
+* <https://github.com/lwolf/kube-cleanup-operator.git>
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| annotations | object | `{}` |  |
+| args[0] | string | `"--namespace=default"` |  |
```

Comment:
List of args could change at any time, and it doesn't require specific order, so may be change this to something like:
```
args | array | list of cli arguments for the binary, like `--namespace=default`, consult README for the list of arguments
```

Reply:
done